libipl/p10/common.C

```diff
@@ -196,3 +198,22 @@ bool ipl_check_functional_master(void)
 
 	return true;
 }
+
+void  log_sbe_ffdc(struct pdbg_target *pib)
+{
+	uint8_t* ffdc = NULL;
+	uint32_t status = 0;
+	uint32_t ffdc_len = 0;
+
+        int ret = sbe_ffdc_get(pib, &status, &ffdc, &ffdc_len);
+	if (!ret)
+        ipl_log(IPL_INFO, "%s status:0x%x ffdc_len:0x%x : Data: ",
+		pdbg_target_path(pib), status, ffdc_len);
+        for (uint32_t i=0; i < ffdc_len; i++)
+		ipl_log(IPL_DEBUG, "%02x", ffdc[i]);
+	ipl_log(IPL_DEBUG,"\n");
```

Comment:
Please indent using tabs, so the code looks consistent.

Reply:
Ack